easybuild/easyconfigs/p/PyYAML/PyYAML-5.3-GCCcore-9.3.0-Python-3.8.2.eb

```diff
@@ -0,0 +1,34 @@
+easyblock = 'PythonPackage'
+
+name = 'PyYAML'
+version = '5.3'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = "https://github.com/yaml/pyyaml"
+description = """PyYAML is a YAML parser and emitter for the Python programming language."""
+
+toolchain = {'name': 'GCCcore', 'version': '9.3.0'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['e9f45bd5b92c7974e59bcd2dcc8631a6b6cc380a904725fce7bc08872e691615']
+
+builddependencies = [('binutils', '2.34')]
+
+dependencies = [
+    ('Python', '3.8.2'),
+    ('libyaml', '0.2.2'),
+]
+
+use_pip = True
+download_dep_fail = True
+sanity_pip_check = True
+
+options = {'modulename': 'yaml'}
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

Comment:
This is the default for a PythonPackage please remove the whole sanity_check_paths section

Reply:
done